local.tf

```diff
@@ -44,6 +44,7 @@ locals {
     enabled_metrics               = ""                              # A comma delimited list of metrics to be collected i.e. GroupMinSize,GroupMaxSize,GroupDesiredCapacity
     placement_group               = ""                              # The name of the placement group into which to launch the instances, if any.
     service_linked_role_arn       = ""                              # Arn of custom service linked role that Auto Scaling group will use. Useful when you have encrypted EBS
+    termination_policies          = "Default"                       # A comma delimited list of policies to decide how the instances in the auto scale group should be terminated. 
```

Comment:
Can we just have this default to `""` like all the other options? Otherwise we get unwanted changes:

```
  ~ module.xxxx.aws_autoscaling_group.workers_launch_template
      termination_policies.#: "0" => "1"
      termination_policies.0: "" => "Default"
```

Reply:
Sure thing, sorry didn't think about that.  AWS's default is "Default" so I was setting it explicitly.  I've made the change.